content/working-with-azure-key-vault-in-csharp/index.md

```diff
@@ -0,0 +1,455 @@
+```
+---
+title: **Working with Azure Key Vault in C\#**
+date: "2020-06-11"
+coverImage: "coverImage.png"
+author: "Joydip Kanjilal"
+description: "An article that demonstrates how to work with Azure Key Vault in C#"
+---
+```
+
+## **Introduction**
```

Comment:
Remove the asterisk from all the headings, they are by default bold

Reply:
Done, thanks!